### ISSUE TITLE
[Feature] Suggested hot tags are not tailored for current reach filter

### ISSUE THREAD

**Issue by catch-21** [pubky/pubky-beta#382]
When I click on the search bar I am presented with 'Hot Tags'. These suggestions are globally relevant and can result in "No posts with this tag" is used with a constraining reach filter.

For example, https://github.com/pubky/pubky-app/issues/376 was raised due to the assumption that this search was not working but instead it was the result of none of the user's _friends_ having used this tag in a post.

It would be an improvement to only list Hot Tags that are relevent for the current filter, or, give an indication that the filter must be switched if no results are yielded.

![1](https://github.com/user-attachments/assets/68e20f90-8991-44dc-b5c2-40e1312af3f8)

![2](https://github.com/user-attachments/assets/0487efef-cd0b-47af-bb28-c7ab22bf80d6)

**Comment by flaviomoceri** [comment]
Makes sense, but to do that I think I need help from the backend guys @JeanlChristophe , I only get hottags in general, not based on user filter

But instead of modifying skunk works in my opinion, it makes more sense to focus on Nexus and make sure the requested feature is present there

**Comment by JeanlChristophe** [comment]
Yes agree

**Comment by flaviomoceri** [comment]
@SHAcollision getHotTags based on filter preferences?
low priority
